source/nijigenerate/viewport/common/mesheditor/tools/lasso.d

```diff
@@ -294,29 +225,30 @@ public:
         if (impl.deformOnly)
             transform = impl.transform;
 
-        impl.foreachMirror((uint axis) {
-            vec3[] mirroredPoints = mirrorLassoPoints(impl, axis, lassoPoints);
+//        impl.foreachMirror((uint axis) {
```

Comment:
If selection is always symmetric points, we could consider displaying only a single lasso tool.

Showing multiple lassos for different axes might make the user think that the tool will select different points for each axis.

if we want to support separate lasso handling per axis, keep the function; otherwise, they could be removed.

Reply:
Currently, the lasso implementation only handles preview and does not project lasso vertices onto mirrored axes.

this relied on the `updateMirrorSelected()`. also see https://github.com/nijigenerate/nijigenerate/issues/98.